_organizers/aditya-dhoot.md

```diff
@@ -0,0 +1,10 @@
+---
+github: adityadhoot
+layout: base
+name: Aditya Dhoot
+photo_url: /static/img/organizers/adityad.jpg
+role: Conference Reviewer
```

Comment:
I think the typical term is Program team.

Reply:
fixed